Scripts/demand/trips.py

```diff
@@ -11,6 +11,7 @@ def __init__(self, zone_data):
 
     def calc_demand(self, purpose, impedance):
         prob = self.calc_prob(purpose, impedance)
+        origin_prob = self.calc_origin_prob(purpose, impedance)
```

Comment:
I suggest we create a new purpose, say "sop - source-other-periperal". Then we add an if-clause to self.calc_prob that redirects to the new function self.calc_origin_prob.

Reply:
https://github.com/HSLdevcom/helmet-model-system/pull/40/commits/f50d84ec85ae84a41d6a97ec1f1dee581a81acc4